nimble/host/include/host/ble_uuid.h

```diff
@@ -82,19 +82,25 @@ typedef union {
 
 #define BLE_UUID16_INIT(uuid16)         \
     {                                   \
-        .u.type = BLE_UUID_TYPE_16,     \
-        .value = (uuid16),              \
+        .u = {                          \
+            .type = BLE_UUID_TYPE_16,   \
+        },                              \
+        .value = { uuid16 },            \
```

Comment:
`()` should stay as they were before, they should not be change to `{}` for scalar value